pypdf/_writer.py

```diff
@@ -1246,7 +1244,7 @@ def generate_file_identifiers(self) -> None:
             id2 = self._compute_document_identifier()
         else:
             id1 = self._compute_document_identifier()
-            id2 = id1
+            id2 = ByteStringObject(id1.original_bytes)
```

Comment:
id1 is a ByteStringObject already. So `.original_bytes` just returns id1. Then wrapping it in `ByteStringObject` doesn't do anything, right?